src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/ComboBoxTests.cs

```diff
@@ -0,0 +1,411 @@
+                for (var j = 0; j < NumItems; j++)
+                {
+                    var item = (_randomUtil.GetInt(true) % 26 + 97).ToString();
+                    items[j] = item;
+                    comboBox.Items.Add(item);
+                }
+
+                comboBox.Sorted = true;
+                Array.Sort(items);
+
+                for (int i = 0; i < NumItems; i++)
+                {
+                    bool result = items[i].Equals(comboBox.Items[i]);
+                    Assert.True(result);
+                }
+
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task SelectedIndex_Changes_SelectedItem_And_Fires_OnSelectedIndexChangedAsync()
+        {
+            await RunComboBoxTestAsync((form, comboBox) =>
+            {
+                InitializeItems(comboBox);
+
+                comboBox.Sorted = false;
+                int numEventsExpected = 0;
+                _numEvents = 0;
+                comboBox.SelectedIndexChanged += (x, y) => _numEvents++;
+
+                //change SelectedIndex 10 times
+                for (var i = 0; i < 10; i++)
+                {
+                    int selectedIndex = comboBox.SelectedIndex;
+                    int index = _randomUtil.GetRange(0, NumItems - 1);
+
+                    comboBox.SelectedIndex = index;
+                    Assert.True(comboBox.SelectedIndex == index);
+
+                    if (selectedIndex != index)
+                        numEventsExpected++;
+
+                    Assert.True(comboBox.Items[comboBox.SelectedIndex].Equals(comboBox.SelectedItem));
+                }
+
+                Assert.Equal(_numEvents, numEventsExpected);
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task FindStringAsync()
+        {
+            await RunComboBoxTestAsync((form, comboBox) =>
+            {
+                InitializeItems(comboBox);
+
+                Assert.True(FindItem(comboBox, false));
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task FindStringExactAsync()
+        {
+            await RunComboBoxTestAsync((form, comboBox) =>
+            {
+                InitializeItems(comboBox);
+
+                Assert.True(FindItem(comboBox, true));
+                return Task.CompletedTask;
+            });
+        }
+
+        [WinFormsFact]
+        public async Task Select_Item_By_KeyboardAsync()
+        {
+            await RunComboBoxTestAsync((form, comboBox) =>
+            {
+                comboBox.SelectedIndexChanged += (x, y) => _numEvents++;
+                var numKeyPresses = _randomUtil.GetRange(1, comboBox.Items.Count);
+                var szItems = CreateItems(NumItems);
+
+                Assert.True(AddItems(comboBox, szItems, NumItems));
+
+                //scroll items using the down arrow key
```

Comment:
All comments must be in sentence case
```suggestion
                // Scroll items using the down arrow key
```